Pipfile

```diff
@@ -0,0 +1,27 @@
+[[source]]
+
+url = "https://pypi.python.org/simple"
+verify_ssl = true
+name = "pypi"
+
+
+[packages]
+
+"stripe" = {path = ".", editable = true}
+
+
+[dev-packages]
+
+pytest = ">=3.4"
+pytest-mock = ">=1.7"
+pytest-xdist = ">=1.22"
+pytest-cov = ">=2.5"
+tox = "*"
+detox = "*"
+"flake8" = "*"
```

Comment:
Does "flake8" need to be quoted?

Reply:
Good question! I'm not entirely sure why pipenv wraps some package names in quotes, but I figure it knows what it's doing :)

Reply:
It doesn't need to be quoted (source: https://github.com/toml-lang/toml#keys), but pipenv's toml library (prettytoml) is a bit overzealous and quotes any package name with a digit in it.